db/migrate/20230328171132_add_user_id_from_account.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdFromAccount < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :accounts, :user
```

Comment:
the column for the reference should be owner_id and also be a foreign key to the table users, and it shouldn't be null

Reply:
Sorry I can not see the documentation
Can you share again, please?

Reply:
oh my bad, here https://api.rubyonrails.org/v7.0.4.2/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference